tests/integration/conftest.py

```diff
@@ -534,6 +534,10 @@ def config(self) -> WorkspaceConfig:
             include_databases=self.created_databases,
         )
 
+    @cached_property
+    def tables_crawler(self) -> TablesCrawler:
+        return TablesCrawler(self.sql_backend, self.inventory_database, self.config.include_databases)
```

Comment:
Comment that the implementation can only be used when the DBR is available, which isn't the case when running integration tests.